conference/invoicing.py

```diff
@@ -49,12 +55,20 @@
 """
 
 
+
 ISSUER_BY_YEAR = {
     2016: ACPYSS_16,
     2017: PYTHON_ITALIA_17,
     2018: EPS_18,
 }
 
+LOCAL_CURRENCY_BY_YEAR = {
```

Comment:
for the next iteration (2019) maybe we could use a table "conference" for that, where we will store the conference and the currency, but not important for this PR.